portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,43 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> commentList;
```

Comment:
Consider this:
private final List<String> commentList = new ArrayList<>();

Making it final means the ArrayList object is assigned on initialization and cannot be changed, but you can still add or delete elements in the list.

Reply:
I understand. I can't make this change because as datastore_branch merged with servlet_branch from another pull request, commentList is no longer used/needed in servlet_branch as a global variable, but I will remember this point for future reference.